kafka/1.7/docker-entrypoint.sh

```diff
@@ -39,6 +70,7 @@ fi
 
 : ${PORT:=9092}
 : ${ADVERTISED_PORT:=9092}
+: ${CONTROLLER_PORT:=9093}
```

Comment:
Should not be this var documented?

Reply:
I have decided against it, given the other port vars also aren't documented explicitly. I have added a sentence to the "Ports" part of the README though. I'm actually wondering whether we do need these variables at all, given a user could always map to a different exposed port.